cmd/rook/util/secret.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"os/signal"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/daemon/ceph/client"
+	"github.com/rook/rook/pkg/daemon/ceph/osd/kms"
+	operator "github.com/rook/rook/pkg/operator/ceph"
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SecretCmd defines a top-level utility command which interacts with secret
+var SecretCmd = &cobra.Command{
+	Use:   "secret",
+	Short: "Secret interacts with a given Key Management System and perform actions.",
+	Long: `The secret sub-command helps interacting with Key Management System.
+	It can perform various actions such as retrieving the content of a Key Encryption Key.`,
```

Comment:
Since `secret` in the k8s context means Kubernetes Secret, maybe it would be better to give a different or longer name? E.g., `key-management`?

Reply:
Makes sense.